scripts/utils.py

```diff
@@ -77,3 +77,8 @@ def get_sf_ingestion_arn():
     APP_NAME = os.environ.get("APP_NAME")
     ENV = os.environ.get("ENV", "dev")
     return f"arn:aws:states:{REGION}:{ACCOUNT_ID}:stateMachine:{APP_NAME}-{ENV}-stepfunction-discover"
+
+
+def get_collection_statemachine_name(collection: str) -> str:
+    timestamp = datetime.datetime.utcnow().strftime("%Y%m%d%H%M%S")
+    return f"{collection}-{timestamp}"
```

Comment:
```suggestion
    return f"{collection[:40]}-{timestamp}"
```

the docs said it has to be less than 80 chars, so you might want to truncate the collection name some of them can be long.